packages/core/src/util/txInspector.ts

```diff
@@ -81,20 +84,64 @@ export type MetadataInspector = Inspector<MetadataInspection>;
 export type PoolRegistrationInspector = Inspector<PoolRegistrationInspection>;
 export type PoolRetirementInspector = Inspector<PoolRetirementInspection>;
 
+type ResolvedInput = {
```

Comment:
Any reason not to use TxOut type here (to also resolve datum etc.)?
```suggestion
type ResolvedInput = TxIn & TxOut;
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1057/commits/08dafef4829ec29541ec7ff4c3cbee7a3f3d6494